recipe/multitenancy/multitenancyclaims/claims.go

```diff
@@ -0,0 +1,14 @@
+package multitenancyclaims
+
+import (
+	"github.com/supertokens/supertokens-golang/recipe/session/claims"
+)
+
+type TypeMultitenancyClaimValidators struct {
+	claims.PrimitiveArrayClaimValidators
+	CheckAccessToDomain func(allowedDomain string, maxAgeInSeconds *int64) claims.SessionClaimValidator
+}
+
+var MultitenancyTenantIdClaim *claims.TypeSessionClaim
+var MultitenancyDomainsClaim *claims.TypeSessionClaim
```

Comment:
i think both should be combined into one claim. The tenantId value would exist if the user has a tenantId, and the domain list would exist if the user has defined the GetDomainsForTenantId function

Reply:
Keeping only MultitenancyDomainsClaim, as per our discussion.

Reply:
Also renamed to `AllowDomainsClaim`